docs/how-to-guides/auto-update-flatcar.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Auto-updating Flatcar Container Linux
+weight: 10
+---
+
+## Introduction
+
+Lokomotive supports Flatcar Container Linux only. [Flatcar Linux Update Operator
+(FLUO)](https://github.com/kinvolk/flatcar-linux-update-operator) can keep the operating system
+updated every time there is a new Flatcar release. FLUO is a supported Lokomotive component. This
+guide will show how to install it and disable its auto-update feature for specific nodes.
+
+## Prerequisites
+
+- A Kubernetes cluster accessible via `kubectl`.
+
+## Steps
+
+### Step 1: Disable auto-update on nodes
+
+If you want to update the Flatcar OS of particular nodes manually in a controlled fashion, there is
+a way to disable automatic updates. Disabling updates can come handy when the workloads run by these
+machines are storage related or ingress nodes, where you cannot afford the slightest loss of
+connection.
```

Comment:
"slightest loss of connection." sounds odd to me here. Maybe we can use some simpler expression here? But I cannot come up with any suggestion.

Reply:
Looks good, thanks!